src/applications/debt-letters/components/DebtDetails.jsx

```diff
@@ -1,120 +1,154 @@
+              content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam id felis pulvinar ligula ultricies sollicitudin eget nec dui. Cras augue velit, pellentesque sit amet nisl ut, tristique suscipit sem. Cras sollicitudin auctor mattis."
+              status="info"
+            />
+            <AdditionalInfo triggerText="Why might I have this debt?">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi
+              excepturi fugit non sunt. Asperiores autem error ipsam magnam
+              minus modi nam obcaecati quasi, ratione rem repellendus
+              reprehenderit ut veritatis vitae.
+            </AdditionalInfo>
+            <OnThisPageLinks isDetailsPage />
 
-          <h2 className="vads-u-font-size--h3 vads-u-margin-y--0">
-            Debt history
-          </h2>
-          <p className="vads-u-margin-y--2">
-            You can view the status or download the letters for this debt.
-          </p>
-          <p className="vads-u-margin-top--0">
-            <strong>Note:</strong> The content of the debt letters below may not
-            include recent updates to your debt reflected above. If you have any
-            questions about your debt history, please contact the Debt
-            Management Center at{' '}
-            <a href="tel: 800-827-0648" aria-label="800. 8 2 7. 0648.">
-              800-827-0648
-            </a>
-            {'.'}
-          </p>
-          {reverse(selectedDebt.debtHistory).map((debtEntry, index) => (
-            <div
-              className="vads-u-display--flex vads-u-flex-direction--column vads-u-margin-y--1p5 vads-u-border-bottom--3px vads-u-border-color--gray-lightest"
-              key={`${debtEntry.letterCode}-${index}`}
+            <h2
+              id="debtLetterHistory"
+              className="vads-u-font-size--h3 vads-u-margin-top--5"
             >
-              <h3 className="vads-u-margin-top--1">
-                {moment(debtEntry.date).format('MMMM D, YYYY')}
-              </h3>
-              <p className="vads-u-font-weight--bold vads-u-margin-y--0">
-                {debtEntry.status}
-              </p>
-              <p className="vads-u-margin-y--1">{debtEntry.description}</p>
-              {/* ToDo: Add link to actual debt letter download in VBMS */}
-              <a className="vads-u-margin-bottom--3" href="#">
-                Download and print the letter
+              Debt letter history
+            </h2>
+            <p className="vads-u-margin-y--2">
+              You can view the status or download the letters for this debt.
+            </p>
+            <p className="vads-u-margin-top--0">
+              <strong>Note:</strong> The content of the debt letters below may
+              not include recent updates to your debt reflected above. If you
+              have any questions about your debt history, please contact the
+              Debt Management Center at{' '}
```

Comment:
i see a lot of this throughout the repo to preserve formatting is there a reason we do this instead of using template strings?

Reply:
Not that I am aware of.  I think linter does this automatically